### PR TITLE
Update google profile endpoint

### PR DESCRIPTION
**Pull request by kalw**
- [X] Bugfix (non-breaking change which fixes an issue)

## Proposed changes
Outdated Google profile endpoint changed with new one.

solve #206 
and may also relate to  #228 

## Checklist

- [X] I have read the [contribution guidelines](https://github.com/node-red/node-red-nodes/blob/master/CONTRIBUTING.md)
- [N/A] For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
- [N/A] I have run `grunt` to verify the unit tests pass
- [N/A] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: google/google.js [hunk 1]
```diff
@@ -250,7 +250,7 @@ module.exports = function(RED) {
             delete credentials.callback;
             RED.nodes.addCredentials(node_id, credentials);
             request.get({
-                url: 'https://www.googleapis.com/plus/v1/people/me',
+                url: 'https://www.googleapis.com/userinfo/v2/me',
```
Comment (dceejay) [inline google/google.js]: Does this change break existing users ? The old endpoint still seems to respond
(though I haven't tried authenticating against it)

**Comment by Gerben321** [comment]
I hope this gets merged soon. I get the "auth worked but profile fetching failed" error while I've got the Google+, Calendar, en d People API enabled. Reallllyy annoying. This should fix it when I try a manual request to the new URL.

File: google/google.js (hunk 1 above)
Comment (kalw) [reply google/google.js]: This seems to be the identification part not authentication.
Could try catch to be over sure that does not break anything.

**Comment by dceejay** [comment]
@Gerben321 - as a sanity check - are you able to manually patch the file at your end and see if indeed it fixes it for you ?

**Comment by Gerben321** [comment]
I actually did that yesterday and it did work after restarting NR! So it's working great for me.
________________________________

**Comment by dceejay** [comment]
Great - just need to fix the test before we can merge it.

**Pull request merged**